src/shared/extensions/ssh.ts

```diff
@@ -143,3 +149,132 @@ export async function startVscodeRemote(
 
     await new ProcessClass(vscPath, ['--folder-uri', workspaceUri]).run()
 }
+
+export abstract class VscodeRemoteSshConfig {
```

Comment:
This code is for "vanilla ssh" so we should probably move it somewhere like `shared/ssh.ts`

```suggestion
export abstract class SshConfig {
```